.github/workflows/docker.yml

```diff
@@ -47,3 +47,5 @@ jobs:
         platforms: linux/amd64,linux/arm64/v8
         tags: ${{ steps.meta.outputs.tags }}
         labels: ${{ steps.meta.outputs.labels }}
+        build-args:
+          - VERSION=${{ steps.meta.outputs.tags }}
```

Comment:
unfortunately this will expand do something like: `ghcr.io/flatcar-linux/ct:v0.9.3 ghcr.io/flatcar-linux/ct:latest`
  ```